samples/interceptors/wso2-micro-integrator/README.md

```diff
@@ -0,0 +1,67 @@
+# Sample Interceptor - WSO2 Micro Integrator
+
+The interceptor service is generated using the interceptor service Open API definition.
+[Swagger Editor](https://editor.swagger.io/) is used to generate the spring service.
```

Comment:
?? Spring service?

Reply:
And this is not generated through [Swagger Editor](https://editor.swagger.io/) right? This readme file is added from auto-generated code in the spring sample.

Reply:
corrected it